templates/project/project.yml

```diff
@@ -121,11 +125,37 @@ Resources:
         PolicyDocument:
           Version: "2012-10-17"
           Statement:
-              - Sid: HostedZoneChangeRecord
+              - Sid: HostedZoneReadRecords
                 Effect: Allow
                 Action:
-                  - route53:*
+                  - route53:Get*
+                  - route53:List*
                 Resource: "*"
+              - Sid: HostedZoneUpdate
```

Comment:
`HostedZoneRead` instead of `HostedZoneUpdate`?

Reply:
I think Update is right because the action is to change the record sets